src/shared/middlewares/authMiddleware.js

```diff
@@ -1,21 +1,25 @@
-import { validateToken } from "../auth/validateToken.js"
+import { validateToken } from "../auth/validateToken.js";
+import { StatusCodes } from "http-status-codes";
 
 const authMiddleware = (req, res, next) => {
+  const token = req.headers["authorization"];
+  if (!token)
```

Comment:
if sem chaves (`{}`) só quando for in-line `if (true) return x`

Reply:
corrigido, o lint alterou rs